README.adoc

```diff
@@ -19,5 +19,25 @@ endif::[]
 
 == Description
 
+You can use the `basic-authentication` policy to manage basic authentication headers sent in API calls. The policy compares the user and password sent in the basic authentication header to an APIM user to determine if the user credentials are valid.
 
-== Configuration
\ No newline at end of file
+To use the policy in an API, you need to:
+
+* create a keyless plan for your API
```

Comment:
the keyless plan is not required. You can use this policy even if you have configured an api-key plan